analyticConfigurations/computed_metric-efs.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
There's no need for this file if there aren't any computed metrics

Reply:
removed